l10n_ch_base_bank/migrations/14.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,7 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
```

Comment:
I campi `res.bank.ccp` e `res.partner.bank.ccp` non sono più presenti in `14.0` quindi forse sono stati spostati o rimossi, come mai non sono gestiti nella migrazione?

Reply:
Perché sono stati integrati nel core di Odoo: [https://github.com/OCA/l10n-switzerland/commit/bebd92b5924b438dbccda59142b33174ce8eb22e](https://github.com/OCA/l10n-switzerland/commit/bebd92b5924b438dbccda59142b33174ce8eb22e)

(vedi il link alla commit anche sul task)

Reply:
> (vedi il link alla commit anche sul task)

Avevo guardato la descrizione dell'attività ma non c'erano riferimenti ai campi `ccp`.

I dati che abbiamo in quei campi in v10, come vengono portati nelle nuove colonne del core, se ci vengono portati?

Reply:
Ho aggiunto il campo allo script (è valorizzato su record che verranno poi eliminati).

Reply:
Come mai il campo `res.bank.ccp` non è gestito nella migrazione?

Reply:
Mi era sfuggito, grazie

Reply:
Se gli unici record di `res.partner.bank` che avevano i campi `bvr_adherent_num` e `ccp` valorizzati sono da eliminare, allora può andare bene eliminare le loro colonne; è un po' rischioso perché da oggi alla migrazione si potrebbero aggiungere record che valorizzano questi campi e ci interessa migrarli, ma per me è sufficiente che @stevech091 ne sia al corrente.

Questo non vuol dire che debba succedere lo stesso per le colonne di `res.bank`.

@stevech091 puoi confermare che non ci interessa portare questi dati in `14.0`?

Reply:
Da come eravamo rimasti quando ci siamo sentiti, questi dati sono dati bancari che non gli servono e che, al massimo, può recuperare tramite fatture, estratti conto ecc...

Per `ccp` di `res.bank` abbiamo un solo record che ha quel campo valorizzato.

Comunque, come dici tu, aspettiamo la conferma definitiva di @stevech091